consensus/istanbul/core/handler.go

```diff
@@ -193,7 +199,26 @@ func (c *core) handleCheckedMsg(msg *message, src istanbul.Validator) error {
 	return errInvalidMessage
 }
 
-func (c *core) handleTimeoutMsg() {
+func (c *core) handleTimeoutMsg(view *istanbul.View) {
+	// TODO-Klaytn-Istanbul: Check whether this statement is required. EN/PN should not handle consensus msgs.
+	if c.backend.NodeType() != common.CONSENSUSNODE {
+		logger.Warn("This is not consensus node, but received an istanbul timeout message",
+			"nodeType", c.backend.NodeType().String())
+		return
+	}
+
+	lastProposal, _ := c.backend.LastProposal()
+	if lastProposal == nil {
+		logger.Error("Cannot find the last proposal")
```

Comment:
Don't we need to leave additional information regarding the received timeout message? (maybe something from `view` variable?)

Reply:
Also "Cannot find the last proposal" seems a bit general. 
How about changing it to "Received timeout message but can't find the last proposal"?

Reply:
Applied, thanks.